### PR TITLE
Feat/93 boas praticas como metodo de buscar lista de usuarios e atualizar lista

### PR DESCRIPTION
**Pull request by pauloarantesmachado**
In this task, best practices were followed to ensure that whenever a person is updated, saved, or deleted, the user list is updated in real-time. This allows the DeviceTrackerRedisServiceImpl class to use an updated list of people, eliminating the need for frequent database requests.

Additionally, a component was created that, upon application startup, makes a request to the database to retrieve a set of people. This approach enhances performance by utilizing Redis and validators.

### CODE REVIEW

File: src/main/java/com/example/geoIot/component/personList/StartupPersonListComponent.java
Comment (AndreWakugawa) [inline src/main/java/com/example/geoIot/component/personList/StartupPersonListComponent.java:1]: `@PostConstruct` annotation is also possible, though it runs AFTER the build and startup stages and not DURING as in the ApplicationRunner interface

File: src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java [hunk 1]
```diff
@@ -16,16 +18,15 @@
...
 import java.util.Set;
 import java.util.stream.Collectors;
 
 @Service
 @EnableScheduling
-public class DeviceTrackerRedisServiceImpl implements DeviceTrackerRedisService {
+public class DeviceTrackerRedisServiceImpl implements DeviceTrackerRedisService{
 
-    private final long MINUTES = 1000 * 60;
+    private final long MINUTES = 1000 * 60 *5;
```
Comment (AndreWakugawa) [inline src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java:29]: MINUTES constant is unintuitive with the new *5 (5 minutes) value

File: src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java [hunk 2]
```diff
@@ -38,23 +39,27 @@ public class DeviceTrackerRedisServiceImpl implements DeviceTrackerRedisService
...
+        if(this.personSet == null){
+            this.personSet = StartupPersonListComponent.getPersons();
+        }
         List<DeviceTrackerRedis> listConverted = this.convertToDeviceTrackerListRedis(pDeviceTrackerRedis,personSet);
         deviceTrackerRedisRepository.saveAll(listConverted);
+        this.synchronizeDataBase();
     }
 
     @Override
     @Scheduled(fixedDelay = MINUTES)
```
Comment (AndreWakugawa) [inline src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java:60]: *5 would be better placed here as in
`@Scheduled(fixedDelay = 5 * MINUTES)`

**Review by AndreWakugawa — CHANGES_REQUESTED** [review]
Changes to `MINUTES` constant in DeviceTrackerRedisServiceImpl

File: src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java (hunk 1 above)
Comment (pauloarantesmachado) [reply src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java]: "The change was made to the minutes constant.

**Pull request merged**